schema/crdb/dbinit.sql

```diff
@@ -3235,7 +3235,19 @@ CREATE TABLE IF NOT EXISTS omicron.public.blueprint (
     -- identifies the latest internal DNS version when blueprint planning began
     internal_dns_version INT8 NOT NULL,
     -- identifies the latest external DNS version when blueprint planning began
-    external_dns_version INT8 NOT NULL
+    external_dns_version INT8 NOT NULL,
+    -- identifies the CockroachDB state fingerprint when blueprint planning began
+    cockroachdb_fingerprint TEXT NOT NULL,
+
+    -- CockroachDB settings managed by blueprints.
+    --
+    -- Settings are typed and not null; we use NULL in these columns to reflect
+    -- that blueprint execution should not modify these options. There is no
```

Comment:
This is a tiny nit, but could we flip the order of this sentence for clarify? Seeing "settings are typed and not null" immediately followed by "we use NULL" confused me at first. Maybe something like

> We use NULL in these columns to reflect that blueprint execution should not modify these options. Applying a setting requires the value to be correctly typed and not NULL. There is no value...